.coveragerc

```diff
@@ -0,0 +1,3 @@
+[report]
+exclude_lines =
```

Comment:
interesting, what else could be added to such a file and what would be useful to us?

Reply:
These lines are not interpreted by by python when running coverage.
Will help to not be frustrated by (<100%) coverage for command line tools